src/main.cpp

```diff
@@ -64,17 +64,3 @@ catch(...)
     std::cin.ignore();
 }
 */
```

Comment:
We should also remove commented code because there's no need to keep it if we have version control.